packages/subject-metadata-controller/src/SubjectMetadataController.ts

```diff
@@ -220,10 +220,10 @@ export class SubjectMetadataController extends BaseControllerV2<
    */
   trimMetadataState(): void {
     this.update((draftState) => {
-      return SubjectMetadataController.getTrimmedState(
+      return SubjectMetadataController.#getTrimmedState(
         // Typecast: ts(2589)
```

Comment:
Does this comment still apply?

Reply:
It does, I had to `@ts-expect-error` to fix the error on this line. Do you have a better solution?